packages/web/src/components/MetaTags.tsx

```diff
@@ -78,22 +78,26 @@ export const MetaTags = (props: MetaTagsProps) => {
     author,
     children,
   } = props
-
+  const appTitle = global.__REDWOOD__APP_TITLE
   return (
     <Head>
       {title && (
         <>
-          <title>{title}</title>
-          <meta property="og:title" content={title} key="title" />
-          <meta property="twitter:title" content={title} />
+          <title>{`${appTitle} | ${title}`}</title>
```

Comment:
That's a good idea !

Reply:
@dthyresson  I one problem `titleTemplate` and many solutions but I don't know wish one to choice. 

So, `titleTemplate` have this template formula : "MySite.com - %s"  `%s` will be replace by the the MyPage (title), but we have noting for SiteName in the Template. Also most of the time you want one template for all in the project. 

Since we want a nice DX, we want a by default convention over configuration, but also give configuration with the better abstraction possible. 

### Solution 1 :
Our own "titleTemplate" as RedwoodProvider props 
```
<RedwoodProvider
titleTemplate = "%MyPage | %SiteName"
>
```
Benefits : Write it once, use it everywhere. 
Downside : need to overwrite titleTemplate if you don't want to use it with title in one page. 

### Solution 2 :
Our own "titleTemplate" as MetaTags props 
```
<MetaTags
  title="${pascalName}"
  description="${pascalName} description"
  titleTemplate = "%MyPage | %SiteName"
/>
```
Benefits : abstract SiteName and can be easily change on every page. 
Downside : Write it on every page

### Solution 3 :
use the "titleTemplate" of helmet and call global everywhere we use it 
```
<Head
  titleTemplate = `%s | ${global.__REDWOOD__APP_TITLE}`
/>
```
Benefits : Simple, can be use everywhere like you want.
Downside : not pretty, less abstraction.

---
Also we probably want a titleTemplate by default for at least `MetaTags` 
Tell me what you think and if you want to talk about it feel free to ping me on Discord 🙂

Reply:
@simoncrypta Thanks for the detailed thoughts ... and I completely see the impact.

In fact, I think let's forget the template idea for now and best to get the community using the generators and getting feedback.

And maybe a template is handy -- maybe not. 

But, I realize no it is not worth holding up this PR. Let's consider these approaches later if needed.

Reply:
FWIW, I like Solution 1 a lot, with some naming changes `%PageTitle | $AppTitle`

Reply:
All right @dthyresson I keep these solutions for maybe another issue after the release and feedback. Also, @dac09 I'm totally  with you for the naming changes.

Reply:
So @dac09 should Simon do option 1? That works for me.